docs/integrations/creating-nodes/build/reference/http-helpers.md

```diff
@@ -0,0 +1,98 @@
+# Making HTTP Requests
+
+!!! note "Programmatic style only"
+	The information in this document is for node building using the programmatic style. It doesn't apply to declarative style nodes.
+
+It's common to call external APIs or make HTTP requests to other services from a node.
+
+This plays a major role during node development, maintenance, and improvements.
+
+n8n provides a flexible helper for making HTTP requests that abstracts away most of the complexity.
```

Comment:
```suggestion
n8n provides a flexible helper for making HTTP requests, which abstracts away most of the complexity.
```

Or "For making HTTP requests, n8n provides a flexible helper that abstracts away most of the complexity."

I'm suggesting this rephrase because the original phrasing confused me for a moment: I read "that" referencing HTTP requests (the nearest object), then saw the verb in singular didn't match the object in plural.